pyrightconfig.stricter.json

```
@@ -62,13 +62,16 @@
         "stubs/ldap3",
         "stubs/m3u8",
         "stubs/Markdown",
+        "stubs/mock",
         "stubs/mysqlclient",
         "stubs/netaddr/netaddr/core.pyi",
         "stubs/netaddr/netaddr/ip/__init__.pyi",
         "stubs/netaddr/netaddr/ip/iana.pyi",
         "stubs/networkx",
         "stubs/oauthlib",
         "stubs/openpyxl",
+        "stubs/opentracing",
```

These are also only a single file change. Would we want to prevent strictness regressions in other files of the package?
```suggestion
        "stubs/mock/mock.pyi",
        "stubs/mysqlclient",
        "stubs/netaddr/netaddr/core.pyi",
        "stubs/netaddr/netaddr/ip/__init__.pyi",
        "stubs/netaddr/netaddr/ip/iana.pyi",
        "stubs/networkx",
        "stubs/oauthlib",
        "stubs/openpyxl",
        "stubs/opentracing/span.pyi",
```